README.md

```diff
@@ -180,9 +180,9 @@ the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory will
 [notes]: #notes
 
 Since the Rust compiler is written in Rust, it must be built by a
-precompiled "snapshot" version of itself (made in an earlier state of
+precompiled "snapshot" version of itself (made in an earlier stage of
 development). As such, source builds require a connection to the Internet, to
-fetch snapshots, and an OS that can execute the available snapshot binaries.
+fetch snapshots and an OS that can execute the available snapshot binaries.
```

Comment:
This was an Oxford comma, which is also correct and makes this more readable, imo

Reply:
@jonas-schievink  Thanks, I have reverted it.